src/common/cups_print.h

```diff
@@ -59,15 +59,12 @@ typedef struct dt_print_info_t
   dt_paper_info_t paper;
 } dt_print_info_t;
 
-// is some printers are available
-gboolean is_printer_available(void);
+// Asynchronous printer discovery, cb will be called for each printer found
```

Comment:
Is this really true? Won't we still block on cupsGetDests until all printers are discovered? Ideally we'd have some form of async way of getting printers from cups itself so that one slow printer doesn't block the discovery of all others. If I'm reading the diff correctly we won't get a single printer until all printers are discovered. That's still much better than before, at least we don't freeze the whole app.